terraform/aks/workspace_variables/production.tfvars.json

```diff
@@ -2,7 +2,7 @@
   "app_environment": "production",
   "cluster": "production",
   "file_environment": "prod",
-  "enable_monitoring": false,
+  "enable_monitoring": true,
```

Comment:
in variables.tf enable_monitoring should default to false and then remove from   "enable_monitoring": false, from test.tfvars.json, preproduction.tfvars.json, and development.tfvars.json,